src/main/java/seedu/duke/Storage.java

```diff
@@ -1,10 +1,113 @@
 package seedu.duke;
 
+import seedu.duke.model.Patient;
+import seedu.duke.model.Record;
+
+import java.io.FileWriter;
+import java.io.File;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.Set;
+import java.util.SortedMap;
+import java.util.TreeMap;
+
 /**
  * This class handles the loading and saving of data from/to a plaintext file on
  * the hard drive.
- * TODO: To be implemented
  */
 public class Storage {
-    
+    protected String filePath;
+
+    /**
+     * This is the constructor of the Storage class.
+     * @param filePath A String of the path to the file that is used for saving/loading
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Saves a SortedMap of data into the file specified by filePath.
+     * @param patientData The data to be written to file
+     * @throws IOException When file is not found etc
+     */
+    public void save(SortedMap<String, Patient> patientData) throws IOException {
+        try {
+            File inFile = new File(filePath);
+            inFile.createNewFile();
```

Comment:
You could make this into a conditional, in case the file already exists.

Reply:
Since the saving has to be done regardless of whether the file exists (if it does, overwrite the existing file, if it doesn't create the file) would it be fine without the conditional?